ravenframework/CustomDrivers/DriverUtils.py

```diff
@@ -98,11 +98,15 @@ def setupCpp():
     @ Out, None
   """
   frameworkDir = findFramework()
-  from utils import utils
-  utils.find_crow(frameworkDir)
-  utils.add_path(os.path.join(frameworkDir,'contrib'))
-  ##TODO REMOVE PP3 WHEN RAY IS AVAILABLE FOR WINDOWS
-  utils.add_path_recursively(os.path.join(frameworkDir,'contrib','pp'))
+  ravenDir = os.path.dirname(frameworkDir)
+  if any(os.path.normcase(sp) == os.path.join(ravenDir, 'crow') for sp in sys.path):
```

Comment:
crow actually is just the c++ build files and so will not be in the path.  Also, find_crow does check to see if crow can be imported before adding to the path.